README.md

```diff
@@ -134,3 +134,17 @@ Please submit [bugs or enhancements requests](https://github.com/vmware-tanzu/co
 More information about troubleshooting and our triage process is available [here](https://tanzucommunityedition.io/docs/main/trouble-faq/).
 
 Information about our roadmap is available [here](https://github.com/vmware-tanzu/community-edition/blob/main/ROADMAP.md).
+
+---
+
+## Join the Community and Make Tanzu Community Edition Better
+
+Tanzu Community Edition Better is better because of our contributors and maintainers. It is because of you that we can bring great software to the community. Please join us during our online community meetings. Details can be found on our [website](https://tanzucommunityedition.io/community/).
```

Comment:
Looks like an extra "Better" in there? :)

Reply:
SO MUCH BETTER! good catch :)